app/services/ccms/requestors/applicant_add_requestor.rb

```diff
@@ -42,6 +42,7 @@ def client(xml)
       def name(xml)
         xml.__send__(:"common:Surname", applicant.last_name)
         xml.__send__(:"common:FirstName", applicant.first_name)
+        xml.__send__(:"common:SurnameAtBirth", applicant.surname_at_birth.presence)
```

Comment:
Do we need to user `.presence` here give the applicant method `surname_at_birth` does this?